build/golang-build/Dockerfile

```diff
@@ -17,14 +18,15 @@ RUN mkdir release && \
     git clone --branch main https://github.com/protocolbuffers/protobuf /go/src/github.com/protocolbuffers/protobuf
 
 RUN export GO111MODULE=on && \
-    go get -u github.com/golang/protobuf/protoc-gen-go && \
+    go install github.com/golang/protobuf/protoc-gen-go@latest && \
+    go install github.com/openconfig/ygot/generator@latest
+
+RUN export GO111MODULE=off && \
     go get -u google.golang.org/protobuf/proto && \
     go get -u golang.org/x/lint/golint && \
     go get -u github.com/fzipp/gocyclo && \
     go get -u github.com/client9/misspell/cmd/misspell && \
-    go get -u github.com/gordonklaus/ineffassign && \
-    go get -u github.com/openconfig/ygot/generator
```

Comment:
I thought YGOT was needed - maybe that was before `model-compiler-docker` came along.
I can't seem to find its use now.
I think we wanted to use the `generate` command through a docker image

Reply:
Ygot is there, it just gets installed above via `go install` because of changes to 1.18

See line 22

Reply:
perfect